src/V3AstNodeExpr.h

```diff
@@ -1285,7 +1285,7 @@ class AstExprStmt final : public AstNodeExpr {
     // METHODS
     string emitVerilog() override { V3ERROR_NA_RETURN(""); }
     string emitC() override { V3ERROR_NA_RETURN(""); }
-    bool cleanOut() const override { return true; }
+    bool cleanOut() const override { return false; }
```

Comment:
I think we should instead ensure that the function inside the ExprStmt e.g. what makes resultp() is always clean.

Otherwise we will be adding a clean in many cases where it is not needed, harming performance.

If you have cases where the clean was incorrect (presumably or you wouldn't have changed it) perhaps split the pre-pull to fix that into a separate pre-pull.